### DISCUSSION TITLE
Unable to change permission on "Status" event subscription

### DISCUSSION THREAD

**Discussion by nick0602** [community/community#24573] [Apps, API and Webhooks]
<p>Hi all, </p>
<p>I’m currently developing an internal Github App for our organization and everything has been working out great so far.</p>
<p>Today I wanted to add the <strong>Status</strong>  event subscription to the app in order to implement some logic related to the corresponding webhooks, however I cannot click the <strong>Status</strong>  checkbox in the settings page of my app under <strong>Subscribe to events</strong>. When I try to, an error message appears on the right hand side saying <strong>Change privileges to be able to select events</strong>.</p>
<p>I tried to give access to pretty much everything on another sample app to see if I missed some permissions but I can’t find any solution… Any idea what the issue could be?</p>
<p>Thanks in advance.</p>

**Comment by Selindek** [comment]
<p>I have the exact same problem. Did you find any solution?</p>

**Comment by nick0602** [comment]
<p>Unfortunately not at the moment. However I contacted GH support and they said they’re investigating but there’s no ETA, so I assume it’s something on their end.</p>
<p>I postponed the changes that required permission changes as I don’t really want to recreate the app, so I hope it gets fixed soon.</p>
<p>I’ll post here when I get updates :+1: </p>

**Comment by git-interview-app** [comment]
<p>Same problem here</p>

**Comment by nick0602** [comment]
<p>I might have found a solution that seems to be working for me for now, feel free to give it a try.</p>
<p><strong>Note that I got the permissions updated notification on my repos, so keep in mind that your users might get the permission updated alert as well while doing this.</strong></p>
<ul>
<li>
<p>Go to your app permissions and events page</p>
</li>
<li>
<p>Make sure the permission related to your event - at the top of the page under “Repository permissions” - is disabled (e.g. Pull Request for the Pull Request Review event). If it’s enabled temporarily disable it.</p>
</li>
<li>
<p>Save changes</p>
</li>
<li>
<p>Go back to permissions and events</p>
</li>
<li>
<p>Enable the permission again</p>
</li>
<li>
<p>DO NOT save changes, but scroll down and look for the event you want your app to be subscribed to </p>
</li>
<li>
<p>Select it and now it should be working</p>
</li>
<li>
<p>Save changes</p>
</li>
</ul>

**Comment by nick0602** [comment, marked as answer]
<p>GH support sent me an email last night stating they fixed it.</p>
<p>I can confirm it’s working as expected now :+1:</p>
